eQuiz/modules/eQuiz.Web/Areas/Admin/Scripts/equiz.layoutController.js

```diff
@@ -0,0 +1,27 @@
+﻿(function (angular) {
+    angular.module('equizModule').controller('LayoutController', function ($scope, $http, $sce) {
```

Comment:
Use this style to define controllers: https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#getters